sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
 
+- Classes marked as final no longer have virtual methods.
```

Comment:
no longer introduce new virtual methods

Reply:
This comment is confusing. It's a statement about the behavior of the compiler, not a statement about the source code. Why are we including information about the behavior of the compiler in our changelog?

Reply:
I can remove it. It isn't relevant to the end customer anyway.